types/sector_class.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+// SectorClass represents the miner's chosen sector size and PoSt/seal proof
+// partitions.
+type SectorClass struct {
```

Comment:
I'm feeling the love for this abstraction.

Reply:
Nice!